### PR TITLE
Issue 187

### PR DESCRIPTION
**Pull request by sharkovadarya**
Lambda expression cannot be used in some rule section https://github.com/JetBrains-Research/snakecharm/issues/187

### CODE REVIEW

**Comment by iromeo** [comment]
Please group arg names in completion list + let's assign some icon, e.g. PlatformIcons.PARAMETER_ICON

File: src/main/resources/SnakemakeBundle.properties [hunk 1]
```diff
@@ -42,6 +42,7 @@ INSP.NAME.only.n.parameters.in.section=Don''t use more than {0} lambda parameter
 INSP.NAME.wildcards.first.parameter='wildcards' has to be the first lambda parameter.
 INSP.NAME.wildcards.first.parameter.preferable=Snakemake documentation suggests it's preferable to name the first parameter 'wildcards'.
 INSP.NAME.non.wildcards.param.first.parameter=''{0}'' cannot be the first lambda parameter in ''{1}'' section.
+INSP.NAME.callables.not.allowed.in.section=Callables are not allowed in ''{0}'' section.
```
Comment (iromeo) [inline src/main/resources/SnakemakeBundle.properties]: Let's replace `callables are not allowed` with smth more user-friendly.  E.g. smth like 'String value is expected ..' or `Function object cannot be a value in ...` or etc

File: src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkLambdaRuleParamsInspection.kt [hunk 2]
```diff
@@ -13,13 +13,25 @@ import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpointArgsSection
 class SmkLambdaRuleParamsInspection : SnakemakeInspection() {
     companion object {
         const val WILDCARDS_LAMBDA_PARAMETER = "wildcards"
-        val ALLOWED_IN_PARAMS = listOf(
+        private const val ATTEMPT_LAMBDA_PARAMETER = "attempt"
```
[the remark this answers is not in the document]
Comment (sharkovadarya) [reply src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkLambdaRuleParamsInspection.kt]: done

File: src/main/resources/SnakemakeBundle.properties (hunk 1 above)
Comment (sharkovadarya) [reply src/main/resources/SnakemakeBundle.properties]: done

**Pull request merged**